packages/react/src/modules/in-app-message/components/Banner/Banner.tsx

```diff
@@ -138,9 +138,15 @@ const DismissButton: React.FC<React.ComponentPropsWithRef<"button">> = ({
 const DefaultView: React.FC<{
   content: BannerContent;
   colorMode?: ColorMode;
-}> = ({ content, colorMode = "light" }) => {
+  onInteracted?: () => void;
+  onDismissClick?: React.MouseEventHandler<HTMLButtonElement>;
+}> = ({ content, colorMode = "light", onInteracted, onDismissClick }) => {
   return (
-    <Root data-knock-color-mode={colorMode}>
+    <Root
+      data-knock-color-mode={colorMode}
+      onClick={onInteracted}
+      onFocus={onInteracted}
```

Comment:
Maybe leave a TODO to discuss w/ Chris+Sam what user interaction should actually fire onInteracted? For example, it looks like the feed component currently fires it when clicking on either the root container or one of the buttons, but not onFocus. Should we be firing onInteracted() on button clicks too?